pycodestyle.py

```diff
@@ -56,6 +56,15 @@
 import time
 import tokenize
 import warnings
+try:
+    from functools import lru_cache
+except ImportError:
+    def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
```

Comment:
As long as `tox.ini` tells 2.6 and 2.7, I'll keep this try, it can easily be dropped later with the drop of 2.x support. I bet some other compatibility things will drop too in any way.